### PR TITLE
Added models: Categories, Comments

### PR DESCRIPTION
**Pull request by evrusha**

### CODE REVIEW

**Review by ritamma — APPROVED** [review]

File: app/models/category.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Category < ApplicationRecord
+  has_many :dreams, dependent: :nullify
+  enum :status, %i[active archived]
```
Comment (fooooxmr) [inline app/models/category.rb:3]: enum :status, {active:0,  archived: 1}

File: db/migrate/20230413133316_create_categories.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreateCategories < ActiveRecord::Migration[7.0]
+  def change
+    create_table :categories do |t|
+      t.string :name
+      t.string :description
+      t.string :image
```
Comment (fooooxmr) [inline db/migrate/20230413133316_create_categories.rb:6]: remove image

**Review by fooooxmr — APPROVED** [review]

File: app/models/category.rb (hunk 1 above)
Comment (evrusha) [reply app/models/category.rb:3]: Fixed

File: db/migrate/20230413133316_create_categories.rb (hunk 2 above)
Comment (evrusha) [reply db/migrate/20230413133316_create_categories.rb:6]: fixed

**Pull request merged**